airbyte-webapp/src/pages/ConnectionPage/pages/ConnectionItemPage/ConnectionPageTitle.module.scss

```diff
@@ -1,9 +1,11 @@
-@use "../../../../scss/colors";
-@use "../../../../scss/variables";
+@use "scss/colors";
+@use "scss/variables";
+@use "scss/_z-indices";
 
 .container {
   text-align: center;
   padding: variables.$spacing-lg variables.$spacing-xl;
+  z-index: z-indices.$connectionPageTitle;
```

Comment:
this resolves the bug we saw in the bug bash where users were unable to click various tabs or click the review button with the backdrop present.  it was because we're using absolute positioning to keep octavia from scrolling.

Reply:
Intead of adding more complexity to the z-index. One alternative I experimented with was not to have the backdrop be fixed but instead I chagned these rules to the backdrop css:

* remove the justify-content: center
* remove the bottom rule
* add 120 px margin top
* let it scroll with the page

This works well with what we have available with the layout. We can revisit this later when we introduce a layout that is maxed to the height of the page.

Reply:
Agree.  That was close to what I had in place before the chat with Nico earlier this week.  I will switch that over.